js/main.js

```diff
@@ -0,0 +1,206 @@
+const $ham = document.querySelector('.fa-bars');
+const $hamMenu = document.querySelector('.ham-menu');
+const $hamSet = document.querySelector('.ham-set');
+const $setList = document.querySelector('.set-list');
+const $back = document.querySelector('.ham-back');
+const $ul = document.querySelector('ul');
+const $main = document.querySelectorAll('.main2');
+const $setImg = document.querySelectorAll('.set-img');
+let cardData;
+let pokeCount = 8;
+let pokeIndex = 8;
+
+// const $pokeInfo = document.querySelector('.poke-info');
+// const $pokeDeck = document.querySelector('.poke-deck');
+
+// search bar focus in and out
+$questionMark.addEventListener('click', function (e) {
+  $searchBarRow.classList.remove('hidden');
+  $searchBar.style.opacity = 1;
+  $searchBar.focus();
+});
+$searchBar.addEventListener('focusout', function (e) {
+  $searchBarRow.classList.add('hidden');
+  $searchBar.style.opacity = 0;
+});
+
+// view swap
+function viewSwap(view) {
+  for (let i = 0; i < $main.length; i++) {
+    if ($main[i].classList.contains(view) !== true) {
+      $main[i].classList.add('hidden');
+    } else {
+      $main[i].classList.remove('hidden');
+    }
+  }
+}
+
+// render pokemon onto search screen
+function renderPokeSearch(pokemon) {
+  // create content and give them names
+  const $img = document.createElement('img');
+  $img.setAttribute('src', pokemon);
+  $img.className = 'column-25 found-poke';
+
+  // appending
+  return $img;
+}
+
+// make the http request
+function searchPoke(name) {
+  const xhr = new XMLHttpRequest();
+  const targetUrl = encodeURIComponent('https://api.pokemontcg.io/v2/cards/?q=name:' + name + '*&pageSize=32');
+  xhr.open('GET', 'https://lfz-cors.herokuapp.com/?url=' + targetUrl);
+  xhr.setRequestHeader('X-Api-Key', 'f81270c6-9d17-41a7-90ff-04e77b2b4273');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    cardData = xhr.response;
+    const $found = document.querySelectorAll('.found-poke');
+    if ($found.length === 0) {
+      for (let i = 0; i < 8; i++) {
+        const image = renderPokeSearch(cardData.data[i].images.large);
+        image.setAttribute('data-cardId', cardData.data[i].id);
+        $pokeSearch.appendChild(image);
+      }
+    } else {
+      for (let i = 0; i < 8; i++) {
+        $found[i].src = cardData.data[i].images.large;
+        $found[i].setAttribute('data-cardId', cardData.data[i].id);
+      }
+    }
+  });
+  xhr.send();
+}
+
+// get info from form and populate page
+$form.addEventListener('submit', function (e) {
+  event.preventDefault();
+  const $value = $form.elements[0].value;
+  searchPoke($value);
+  viewSwap('poke-search-div');
+  uiControlSwap('search');
+  $searchBar.blur();
+});
+
+// arrows can move forward or backwards for searched cards;
+$rightArrow.addEventListener('click', function (e) {
+  if (cardData.length !== 0) {
+    const foundPoke = document.querySelectorAll('.found-poke');
+    let j = 0;
```

Comment:
What is the point of this variable? `j` isn't very descriptive of what this is supposed to be doing. Are you just using this to keep track / count? If so, a better name would be `count`